src/main/scala/scorex/api/http/leasing/LeaseRequest.scala

```diff
@@ -15,5 +16,21 @@ case class LeaseRequest(@ApiModelProperty(value = "Base58 encoded sender public
                         recipient: String)
 
 object LeaseRequest {
-  implicit val leaseCancelRequestFormat: Format[LeaseRequest] = Json.format
-}
+  val leaseRequestReads: Reads[LeaseRequest] = (
+    (__ \ "sender").read[String] and
+      ((__ \ "amount").read[Long] | (__ \ "amount").read[String].map(_.toLong)) and
```

Comment:
the indentation should be 2 spaces

Reply:
yes, but intellij just auto generated this style, fixed in the next commit

Reply:
OK